### PR TITLE
Added fakenick feature

### PR DESCRIPTION
**Pull request by good-live**
needs to get tested

### CODE REVIEW

File: addons/sourcemod/scripting/ttt/ttt_fakenick.sp [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+	Menu menu = new Menu(SelectMenu_Callback);
+	
+	char sTitle[64];
+	Format(sTitle, sizeof(sTitle), "%T", "FakeNick Title", client);
+	
+	menu.SetTitle(sTitle);
+	
+	int c = 0;
+	char sName[MAX_NAME_LENGTH];
+	for (int i = 1; i <= MaxClients; i++)
```
Comment (Bara20) [inline addons/sourcemod/scripting/ttt/ttt_fakenick.sp]: Line 136-138 könnte auch mit "LoopValidClients(i)" ersetzt werden.

**Comment by Bara20** [comment]
Tested or atm. just coded?

**Comment by good-live** [comment]
Currently it's not working. Need to take a look at it again.

**Comment by Bara20** [comment]
Closed - No changes over months.

**Pull request closed without merging**